__tests__/components/Videos/Upload/index.test.tsx

```
@@ -0,0 +1,125 @@
+    error: vi.fn(),
+    success: vi.fn(),
+    loading: vi.fn(),
+  },
+}));
+
+describe('HookMultiStepForm Token Gating', () => {
+  beforeEach(() => {
+    vi.clearAllMocks();
+    
+    // Default mock implementations
+    (useActiveAccount as any).mockReturnValue({ address: '0x123' });
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)